flink-clients/src/main/java/org/apache/flink/client/program/ClusterClient.java

```diff
@@ -101,17 +101,6 @@ public ClusterClient(Configuration flinkConfig) {
 		this.timeout = AkkaUtils.getClientTimeout(flinkConfig);
 	}
 
-	// ------------------------------------------------------------------------
-	//  Startup & Shutdown
-	// ------------------------------------------------------------------------
-
-	/**
-	 * User overridable hook to close the client, possibly closes internal services.
-	 */
-	public void shutdown() throws Exception {
```

Comment:
As I have discussed with @tillrohrmann in FLINK-13750, maybe it is also reasonable we have a empty impl for `close` in `ClusterClient` and let only those who actually requires cleanups override it.